src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -71,6 +79,17 @@ class AddressLookUpController extends AbstractController {
     };
   }
 
+  private isAddressListRequired(pageType: string): boolean {
+    return pageType === AddressLookUpPageType.chooseRegisteredOfficeAddress;
+  }
+
+  private async getAddressList(tokens, postcode: string): Promise<UKAddress[]> {
```

Comment:
this private function doesn't add anything so will change to call addressService instead of this function